make build failed

Made some page and blog updates for KubeVirt Summit and wanted to check the local build. unfortunately it failed out.

To repeat:
- $ export SELINUX_ENABLED=true
- ran `make build_img`
- `make build`

Output: 
Makefile: Build jekyll site
scripts/update_changelog.sh
Cloning into 'build/kubevirt'...
remote: Enumerating objects: 245431, done.
remote: Counting objects: 100% (181/181), done.
remote: Compressing objects: 100% (117/117), done.
remote: Total 245431 (delta 84), reused 120 (delta 63), pack-reused 245250
Receiving objects: 100% (245431/245431), 266.44 MiB | 4.50 MiB/s, done.
Resolving deltas: 100% (168708/168708), done.
Already on 'main'
Your branch is up to date with 'origin/main'.
Already up to date.
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
...
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
egrep: warning: egrep is obsolescent; using grep -E
rake
make: rake: No such file or directory
make: *** [Makefile:105: build] Error 127

I have not looked further into it.

--- a/pages/quickstart_cloud.md
+++ b/pages/quickstart_cloud.md
@@ -45,7 +45,7 @@ KubeVirt can be installed using the KubeVirt operator, which manages the lifecyc
   ```bash
   export VERSION=$(curl -s https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
   echo $VERSION
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml"
   ```
 
   > warning "Nested virtualization"

--- a/pages/quickstart_cloud.md
+++ b/pages/quickstart_cloud.md
@@ -59,7 +59,7 @@ KubeVirt can be installed using the KubeVirt operator, which manages the lifecyc
 * Again use `kubectl` to deploy the KubeVirt custom resource definitions:
 
   ```bash
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml"
   ```
 
 ### Verify components

--- a/pages/quickstart_kind.md
+++ b/pages/quickstart_kind.md
@@ -59,7 +59,7 @@ KubeVirt can be installed using the KubeVirt operator, which manages the lifecyc
 * Again use `kubectl` to deploy the KubeVirt custom resource definitions:
 
   ```bash
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml"
   ```
 
 ### Verify components

--- a/pages/quickstart_kind.md
+++ b/pages/quickstart_kind.md
@@ -45,7 +45,7 @@ KubeVirt can be installed using the KubeVirt operator, which manages the lifecyc
   export VERSION=$(curl -s https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
 
   echo $VERSION
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml"
   ```
 
   > warning "Nested virtualization"

--- a/pages/quickstart_minikube.md
+++ b/pages/quickstart_minikube.md
@@ -104,7 +104,7 @@ Below are two examples of how to install KubeVirt using the latest release.
 * Again use `kubectl` to deploy the KubeVirt custom resource definitions:
 
   ```bash
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-cr.yaml"
   ```
 
 ### Verify components

--- a/pages/quickstart_minikube.md
+++ b/pages/quickstart_minikube.md
@@ -90,7 +90,7 @@ Below are two examples of how to install KubeVirt using the latest release.
   ```bash
   export VERSION=$(curl -s https://storage.googleapis.com/kubevirt-prow/release/kubevirt/kubevirt/stable.txt)
   echo $VERSION
-  kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml
+  kubectl create -f "https://github.com/kubevirt/kubevirt/releases/download/${VERSION}/kubevirt-operator.yaml"
   ```
 
 > warning "Nested virtualization"

